test/feature/step_definitions/setup_definitions.coffee

```diff
@@ -34,3 +34,7 @@ module.exports = ->
     @createFile "routes.coffee", routesFileContents, (err) =>
       return done.fail err if err
       done()
+
+
+  @Then /^the app doesn't error$/, (done) ->
```

Comment:
I think when I do my refactor of lazy boot up then this step definition will cease to be empty